v2/cmd/nuclei/main.go

```diff
@@ -32,6 +36,24 @@ func main() {
 
 	readConfig()
 
+	// Profiling related code
+	if memProfile != "" {
+		f, err := os.Create(memProfile)
+		if err != nil {
+			log.Fatalf("profile: could not create memory profile %q: %v", memProfile, err)
```

Comment:
I'd suggest using `gologger` instead of `log` as it might interfere with the standard log suppression we introduced at https://github.com/projectdiscovery/nuclei/blob/e13939cdfe9f8fdc3035a81fb07a8f13347b2f33/v2/internal/runner/options.go#L156